software/winrm/pom.xml

```diff
@@ -44,6 +43,12 @@
             <groupId>io.cloudsoft.windows</groupId>
             <artifactId>winrm4j</artifactId>
             <version>${winrm4j.version}</version>
+            <exclusions>
+                <exclusion>
+                    <groupId>org.slf4j</groupId>
+                    <artifactId>*</artifactId>
+                </exclusion>
+            </exclusions>
```

Comment:
When I add to software/winrm/pom.xml:

    <!-- Excluded from winrm4j the wrong jcl-over-slf4j version; include the right one -->
    <dependency>
        <groupId>org.slf4j</groupId>
        <artifactId>jcl-over-slf4j</artifactId>
        <version>${slf4j.version}</version>
    </dependency>
 
Then the maven-enforcer fails with the error below. Does it not understand exclusions?! Or is it just very very strict?

    [WARNING] 
    Dependency convergence error for org.slf4j:jcl-over-slf4j:1.7.12 paths to dependency are:
    +-org.apache.brooklyn:brooklyn-software-winrm:0.9.0-SNAPSHOT
      +-io.cloudsoft.windows:winrm4j:0.3.1
        +-io.cloudsoft.windows:winrm4j-client:0.3.1
          +-org.apache.cxf:cxf-rt-transports-http-hc:3.1.4
            +-org.slf4j:jcl-over-slf4j:1.7.12
    and
    +-org.apache.brooklyn:brooklyn-software-winrm:0.9.0-SNAPSHOT
      +-org.slf4j:jcl-over-slf4j:1.6.6
    
    [WARNING] Rule 2: org.apache.maven.plugins.enforcer.DependencyConvergence failed with message:
    Failed while enforcing releasability the error(s) are [
    Dependency convergence error for org.slf4j:jcl-over-slf4j:1.7.12 paths to dependency are:
    +-org.apache.brooklyn:brooklyn-software-winrm:0.9.0-SNAPSHOT
      +-io.cloudsoft.windows:winrm4j:0.3.1
        +-io.cloudsoft.windows:winrm4j-client:0.3.1
          +-org.apache.cxf:cxf-rt-transports-http-hc:3.1.4
            +-org.slf4j:jcl-over-slf4j:1.7.12
    and
    +-org.apache.brooklyn:brooklyn-software-winrm:0.9.0-SNAPSHOT
      +-org.slf4j:jcl-over-slf4j:1.6.6
    ]

When I check with `mvn dependency:tree`, it only mentions `org.slf4j:jcl-over-slf4j:jar:1.6.6`.

Any ideas @neykov ?